TQ42-1444 update demo notebook

davidbtq opened the pull request:
Demonstrate how to list the human-readable project names

sbtq reviewed (changes requested):
What are you doing here? Why are you importing the grpc messages instead of just using the python classes for this?

If you want to have the names for projects you can just do it like this:

```python
from tq42.project import list_all
from tq42.client import TQ42Client

with TQ42Client() as client:
  proj_selected = list_all_projects(client=client, organization_id=org_selected.id)[0]
  print(proj_selected.data.name)
```

? Not sure why you are making it so complicated...

davidbtq commented:
Thanks for the tip -- I changed the notebook to use the python classes. I also added a printout for the org and exp names in addition to the project names.

sbtq reviewed (changes requested):
Would be heavily in favor of not defining specific print methods. Our python classes should be descriptive enough with `__str__` & `__repr__` in the future.
